android/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,4 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-
-    package="com.yuanzhou.vlc">
```

Comment:
I've this error:

npx @react-native-community/cli config
error Failed to build the app: **No package name found.** We couldn't parse the namespace from neither your build.gradle[.kts] file at /Users/foo/Documents/source/baz/node_modules/react-native-vlc-media-player/android/build.gradle nor your package in the AndroidManifest at /Users/foo/Documents/source/baz/node_modules/react-native-vlc-media-player/android/src/main/AndroidManifest.xml.